src/reducers/books.js

```diff
@@ -0,0 +1,14 @@
+import { createBook, removeBook } from '../actions/index';
+
+const booksReducer = (state = [], action) => {
+  switch (action.type) {
+    case createBook:
+      return [...state, action.payload];
+    case removeBook:
```

Comment:
**[REQUIRED]**
* [ ] removeBook should filter the book store and return all but the selected book, to achieve this we need to use a method like `.filter` which do not mutate state but returns a copy of the store minus the select book. ie
`  case 'REMOVE_BOOK': return state.filter(book => book.id !== action.id);`